### PR TITLE
[www] Move Plugins page from /packages to /plugins

### LINKED ISSUE #4669: redirect /packages/ to /plugins/
So even though the plugin library includes more than just plugins, we're calling it a plugin library anyway since the term is more common and recognizable. The URL ought to reflect this name.

### PR DESCRIPTION
**Pull request by tsriram**
Resolves #4669

### CODE REVIEW

**Comment by pieh** [comment]
@tsriram it's build automatically, but link isn't displayed - build for your PR is here - https://deploy-preview-4705--gatsbyjs.netlify.com/

**Comment by tsriram** [comment]
Ah, thanks @pieh! I was checking https://deploy-preview-4705--gatsby.netlify.com/ (`gatsbyjs` without js) repeatedly for a long time and thought it's not built automatically :smile:

**Comment by KyleAMathews** [comment]
You've changed all the slugs for the packages to use `plugins`. We only want to change the plugin library to `/plugins/` not all the packages.

**Comment by KyleAMathews** [comment]
Thanks for taking this on!

**Comment by tsriram** [comment]
I reverted the packages' slug to `/packages/`, but I'm not sure if that's right. Doesn't the slug need to `/plugins/` so that the links in search sidebar in Plugins page take you to the right page?

File: www/src/components/searchbar-body.js [hunk 1]
```diff
@@ -310,7 +310,7 @@ const Result = ({ hit, pathname, search }) => {
   return (
     <Link
       to={{
-        pathname: `/packages/${hit.name}/`,
+        pathname: `/plugins/${hit.name}/`,
```
Comment (KyleAMathews) [inline www/src/components/searchbar-body.js]: Revert this

File: www/src/layouts/index.js [hunk 2]
```diff
@@ -31,9 +31,9 @@ class DefaultLayout extends React.Component {
     const isTutorial =
       this.props.location.pathname.slice(0, 10) === `/tutorial/`
     const isFeature = this.props.location.pathname.slice(0, 9) === `/features`
-    const isPackage = this.props.location.pathname.slice(0, 9) === `/packages`
+    const isPackage = this.props.location.pathname.slice(0, 8) === `/plugins`
```
Comment (KyleAMathews) [inline www/src/layouts/index.js]: Also check if the pathname is `/packages/`

File: www/src/layouts/index.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ class DefaultLayout extends React.Component {
     const packageSidebarWidth = rhythm(17)
 
     const leftPadding = rhythmSize => {
-      if (this.props.location.pathname.slice(0, 9) === `/packages`) {
+      if (this.props.location.pathname.slice(0, 8) === `/plugins`) {
```
Comment (KyleAMathews) [inline www/src/layouts/index.js]: check for `/packages/` as well

**Comment by m-allanson** [comment]
@KyleAMathews I think this is ready for another look?

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**